### PR TITLE
Update rocWMMA version

### PR DESCRIPTION
**Pull request by cgmillette**
- Updated the docs URL
- Updated version 
- Updated changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,46 @@
 # Change Log for rocWMMA
 
-Full documentation for rocWMMA is available at [rocwmma.readthedocs.io](https://rocwmma.readthedocs.io/en/latest/).
+Full documentation for rocWMMA is available at [rocm.docs.amd.com/projects/rocWMMA](https://rocm.docs.amd.com/projects/rocWMMA/en/latest).
```
Comment (CongMa13) [inline CHANGELOG.md:3]: Back to docs.amd.com?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,46 @@
...
+
+## (Unreleased) rocWMMA 1.3.0 for ROCm 6.0.0
+
+### Added
+- Added support for gfx940, gfx941 and gfx942 targets
+- Added support for f8, bf8 and xfloat32 datatypes
+- Added support for HIP_NO_HALF, __ HIP_NO_HALF_CONVERSIONS__ and __ HIP_NO_HALF_OPERATORS__ (e.g. pytorch environment)
```
Comment (CongMa13) [inline CHANGELOG.md:10]: seems an extra whitespace. 
__ HIP_NO_HALF_CONVERSIONS__ and 
__ HIP_NO_HALF_OPERATORS__

**Review by CongMa13 — APPROVED** [review]
lgtm. Just need to remove extra spaces.

File: CHANGELOG.md (hunk 1 above)
Comment (cgmillette) [reply CHANGELOG.md:3]: Yes, this is the one that needs to be used

File: CHANGELOG.md (hunk 2 above)
Comment (cgmillette) [reply CHANGELOG.md:10]: The markup formatting interprets as bold

**Review by mkarunan — APPROVED** [review]
LGTM

**Pull request merged**